node/src/chain_spec.rs

```diff
@@ -423,3 +428,84 @@ fn westmint_testnet_genesis(
 		cumulus_pallet_aura_ext: Default::default(),
 	}
 }
+
+pub fn statemine_mainnet_config(id: ParaId) -> StatemineChainSpec {
+	//TODO check if correct
+	let data = r#"
+		{
+			"ss58Format": 2,
+			"tokenDecimals": 12,
+			"tokenSymbol": "ksm"
+		}"#;
+
+	let properties = serde_json::from_str(data).unwrap();
+	StatemineChainSpec::from_genesis(
+		// Name
+		"statemine",
+		// ID
+		"statemine",
+		ChainType::Live,
+		move || {
+			statemine_mainnet_genesis(
+				vec![
+					//TODO replace with proper keys
+					(
```

Comment:
Talking with @lovelaced about this. We may need some guidance, if there are Aura keys we should be using to launch the chain. @bkchr